wordpress-mobile/WordPress-iOS-Shared
# Pin the ruby version for this project

This ensures we’re using a consistent version, which will prevent bundler weirdness.
To Test:
1. See that CI is passing.
2. Check out this branch, and run ruby -v. The output should be something like `ruby 2.6.4p104 (2019-08-28 revision 67798) [x86_64-darwin19]`.
